### PR TITLE
Translation to Spanish and Français

### PR DESCRIPTION
**Pull request by BobbyESP**
I just modified the source code to implement the spanish version. This can be changed modifying the JSON file. By the moment, are only supported Spanish and English.

### CODE REVIEW

File: LogicHandler.cs [hunk 1]
```diff
@@ -358,8 +379,17 @@ private void StartPointRefresh()
 
         private void pointTimer_Tick(object sender, EventArgs e)
         {
-            UpdateLatest();
-            new RankDetection();
+            if(language== "es")
+            {
+                UpdateLatest();
```
Comment (joanbosch) [inline LogicHandler.cs:378]: This line is replied in both cases. Should be refactored and placed above as before.

Comment (BobbyESP) [reply LogicHandler.cs:378]: Ok. Thx. Im gonna see it now, but i think the code is fine, only a GitHub bug

File: LogicHandler.cs [hunk 2]
```diff
@@ -65,11 +67,23 @@ public LogicHandler(ValorantOverStream instance)
...
             login();
 
-            UpdateLatest();
-            new RankDetection();
 
-            StartPointRefresh();
-            StartRELOGTimer();
+            if (language == "es")
+            {
+                UpdateLatest();
```
Comment (joanbosch) [inline LogicHandler.cs:87]: The functions `UpdateLastest()`, `StartPointRefresh()` and `StartRELOGTimer()` can be called outside the if statement.

File: LogicHandler.cs (hunk 1 above)
Comment (BobbyESP) [reply LogicHandler.cs:376]: Ok. Fixed in the new push. Thanks.

**Pull request closed without merging**